wagtail_modeltranslation/patch_wagtailadmin.py

```diff
@@ -358,13 +358,12 @@ def _localized_update_descendant_url_paths(page, old_url_path, new_url_path, lan
         cursor.execute(update_statement, [new_url_path, len(old_url_path) + 1, page.path + '%', page.id])
     else:
         (Page.objects
-            .rewrite(False)
```

Comment:
Hi @timstallmann, 

Did removing this line changed the behaviour of the code for you?

The reason it's there it's so the original `url_path` gets updated when this code is ran by the original `Page.save()` code (when `localized_url_path = 'url_path'`). Even though removing it doesn't break things it ends up changing wagtailcore's default behaviour.

Reply:
Thanks for the clarification on what that line does! I was seeing weird behavior related to url paths so that explains why :)

Replaced that line in the PR, and so now this PR changes the